frontend/templates/header.html

```diff
@@ -3,7 +3,7 @@
 <header class="govuk-header " role="banner" data-module="govuk-header">
     <div class="govuk-header__container govuk-width-container">
         <div class="govuk-header__logo">
-            <a class="govuk-header__link govuk-header__link--homepage">
+            <a class="govuk-header__link govuk-header__link--homepage" style="pointer-events: none;">
```

Comment:
Any idea where this style change has come from? Doesn't seem related to translations? Can we remove it?

Reply:
not sure where that came from 🤔 but have removed this now 👍🏾